exchanges/orders.py

```diff
@@ -0,0 +1,146 @@
+from .assets import Asset
+
+def basic_order_check(action, asset_pair):
+    if action not in ['buy', 'sell']:
+        raise ValueError("Action must be 'buy' or 'sell'")
+
+    asset0, asset1 = asset_pair
+    if not isinstance(asset0, Asset) or not isinstance(asset1, Asset):
+        raise ValueError("Not recognized asset pair")
```

Comment:
This error message is misleading. The problem is not that the pair is not recognized, it is that one of the assets is not an `Asset`. Should be something like 
```
An asset pair must be composed of two instances of Asset, not ({}, {})'.format(class(asset0), class(asset1))
```